rgw/v2/tests/s3_swift/test_cloud_transition.py

```diff
@@ -56,6 +59,20 @@ def test_exec(config, ssh_con):
             )
             log.info("creating bucket with name: %s" % bucket_name_to_create)
```

Comment:
can we include format string instead of %s

Reply:
Done